src/pages/iou/IOUTransactions.js

```diff
@@ -29,22 +29,46 @@ const defaultProps = {
 };
 
 class IOUTransactions extends Component {
+    constructor(props) {
+        super(props);
+
+        this.getActionForTransaction = this.getActionForTransaction.bind(this);
+    }
+
+    /**
+     * Given a transaction from an IOU Report, returns the chatReport action with a matching transactionID. Unless
+     * something has gone wrong with our storing logic, there should always exist an action for each transaction.
```

Comment:
That's a good point. But as I will probably end up throwing away this code with the new IOU issue, I would prefer to not implement this now -- I will definitely bear this in mind which refactoring this logic though. 

It might become even simpler than that, as chatReport actions are going to be made the source of truth here, instead of IOU transactions -- we might not even need the IOUtransactions data (as `ReportTransaction` [does not need anything](https://github.com/Expensify/Expensify.cash/blob/95d175857f86d08bef36e4a7aeee897654e359b6/src/pages/iou/IOUTransactions.js#L67-L72) from the transaction object)

Reply:
No further action to take here. This temporary logic is overwritten by the [next IOU PR here](https://github.com/Expensify/Expensify.cash/blob/8d6177a94012dc2d8f0295b52fe45ce9c3e12b5d/src/pages/iou/IOUTransactions.js#L26-L46), which replaces the action/transaction matching. 

Marking as resolved